entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 python manage.py migrate --noinput
-uwsgi --ini uwsgi.ini --wsgi-file ./onlineweb4/wsgi.py
+uwsgi --ini uwsgi.ini --wsgi-file ./onlineweb4/wsgi.py --socket 0.0.0.0:8000
```

Comment:
You said "Remove hard-coded ports" in the description, but I think this adds it?

Reply:
Ah, you moved it from `uwsgi.ini`